src/Sulu/Component/Webspace/DefaultTemplate.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace;
+
+/**
+ * Container for a default template definition.
+ */
+class DefaultTemplate
+{
+    /**
+     * @var string
+     */
+    private $type;
+
+    /**
+     * @var string
+     */
+    private $template;
+
+    /**
+     * @var string|null
+     */
+    private $parentTemplate;
+
+    /**
+     * DefaultTemplate constructor.
+     *
+     * @param string $type
+     * @param string $template
+     * @param string|null $parentTemplate
```

Comment:
We don't add the `@param` tags if they don't add any value (what you've written here is also already written in the type hints of the constructor). Sorry, could have seen that before :see_no_evil:

Reply:
Ok, removed :)